ecommerce_extensions/payment/processors/edupay.py

```diff
@@ -202,7 +202,7 @@ def _get_owner_parameters(self, owner):
             if hasattr(owner, value):
                 parameters[key] = str(getattr(owner, value))
             elif value in getattr(owner, 'extended_profile_fields', {}):
-                parameters[key] = str(owner.extended_profile_fields['extended_profile_fields'])
+                parameters[key] = str(getattr(owner.extended_profile_fields, value, ''))
```

Comment:
The verification is done in the previous line,  hence this line will only be executed  if value exists so if you want to send the  key with an empty value you have to modify the previous line